client/src/components/Sidebar/Sidebar.js

```diff
@@ -31,15 +31,16 @@ const Sidebar = (props) => {
       {conversations
         .filter((conversation) => conversation.otherUser.username.includes(searchTerm))
         .map((conversation) => {
-          return <Chat conversation={conversation} key={conversation.otherUser.username} />;
+          return <Chat conversation={conversation} key={conversation.otherUser.username} userId={userId}/>;
         })}
     </Box>
   );
 };
 
 const mapStateToProps = (state) => {
   return {
-    conversations: state.conversations
+    conversations: state.conversations, 
+    userId: state.user.id
```

Comment:
Follow the `userId` prop and you'll notice that you're not using it anywhere. Could you confirm if you can safely remove it?

Reply:
removed